styleguide/source/assets/images/logo-reversed.svg

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
do you have a specific thing you use to minify the svgs? I just want to point out that I don't think the other SVGs are minified in the styleguide